candle-flash-attn/build.rs

```diff
@@ -152,18 +166,6 @@ fn main() -> Result<()> {
     println!("cargo:rustc-link-lib=dylib=cudart");
     println!("cargo:rustc-link-lib=dylib=stdc++");
 
-    /* laurent: I tried using the cc cuda integration as below but this lead to ptaxs never
```

Comment:
Could we keep this comment, it's probably useful to know that this was tried if someone wants to pick it up again.